src/Handler/QueryMonitorHandler.php

```diff
@@ -0,0 +1,55 @@
+<?php # -*- coding: utf-8 -*-
+/*
+ * This file is part of the Wonolog package.
+ *
+ * (c) Inpsyde GmbH
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Inpsyde\Wonolog\Handler;
+
+use Monolog\Handler\AbstractProcessingHandler;
+use Psr\Log\LogLevel;
+
+/**
+ * Class QueryMonitorHandler.
+ *
+ * @package BoxUk\Plugins\Base
+ */
+class QueryMonitorHandler extends AbstractProcessingHandler {
+	private static $level_map = array(
```

Comment:
I could prefer a constant more than a static variable. 
Moreover we use short array syntax.

Reply:
Ah for some reason I thought array constants were a PHP 7 thing but it seems not, will chance and use short array syntax 👍

Reply:
Fixed.